### PR TITLE
feat: init bottom bar skeleton, add feature flag for git

### PR DESCRIPTION
**Pull request by rishabhsaxena**
## Description
Init bottom bar skeleton

## Type of change
- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
- Manually

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: init-bottom-bar-skeleton 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 55.03 **(0.02)** | 36.9 **(0.01)** | 33.96 **(0.01)** | 55.58 **(0.02)**
 :green_circle: | app/client/src/constants/DefaultTheme.tsx | 92.92 **(0.06)** | 89.47 **(0)** | 71.43 **(0)** | 91.75 **(0.08)**
 :green_circle: | app/client/src/constants/messages.ts | 70.9 **(-0.01)** | 100 **(0)** | 12.2 **(-0.04)** | 78.89 **(0.05)**
 :red_circle: | app/client/src/pages/Editor/MainContainer.tsx | 100 **(0)** | 75 **(-25)** | 100 **(0)** | 100 **(0)**
 :red_circle: | app/client/src/pages/Editor/WidgetsEditor.tsx | 92.54 **(-0.11)** | 75 **(0)** | 100 **(0)** | 92.19 **(-0.12)**
 :green_circle: | app/client/src/pages/Editor/routes.tsx | 42.03 **(0.6)** | 22.22 **(0)** | 7.14 **(0.47)** | 46.67 **(0.77)**
 :red_circle: | app/client/src/pages/Editor/utils.ts | 42.55 **(0.16)** | 43.08 **(-1.36)** | 25 **(0)** | 44.32 **(0.13)**
 :green_circle: | app/client/src/pages/Editor/APIEditor/CurlImportForm.tsx | 75.76 **(0.76)** | 87.5 **(1.79)** | 33.33 **(0)** | 80 **(0.69)**
 :green_circle: | app/client/src/pages/Editor/APIEditor/Form.tsx | 37.5 **(0.46)** | 43.75 **(0)** | 0 **(0)** | 39.22 **(0.51)**
 :sparkles: :new: | **app/client/src/pages/Editor/BottomBar/index.tsx** | **66.67** | **100** | **0** | **66.67**
 :green_circle: | app/client/src/pages/Editor/DataSourceEditor/JSONtoForm.tsx | 15.69 **(0.21)** | 18.18 **(0)** | 4.17 **(0.32)** | 16.67 **(0.24)**
 :green_circle: | app/client/src/pages/Editor/DataSourceEditor/RestAPIDatasourceForm.tsx | 25.4 **(0.27)** | 18.18 **(0)** | 3.13 **(0.19)** | 28.48 **(0.34)**
 :green_circle: | app/client/src/pages/Editor/IntegrationEditor/ActiveDataSources.tsx | 54.17 **(4.17)** | 75 **(0)** | 0 **(0)** | 56.52 **(4.52)**
 :green_circle: | app/client/src/pages/Editor/IntegrationEditor/IntegrationsHomeScreen.tsx | 27.15 **(0.18)** | 11.9 **(0)** | 4.76 **(0.21)** | 30.3 **(0.22)**
 :green_circle: | app/client/src/pages/Editor/IntegrationEditor/MockDataSources.tsx | 45.95 **(1.21)** | 87.5 **(0)** | 0 **(0)** | 45.95 **(1.21)**
 :green_circle: | app/client/src/pages/Editor/QueryEditor/EditorJSONtoForm.tsx | 40.35 **(0.47)** | 33.8 **(0)** | 0 **(0)** | 43.13 **(0.54)**
 :green_circle: | app/client/src/pages/Editor/SaaSEditor/ListView.tsx | 34.62 **(0.44)** | 32 **(0)** | 6.25 **(0.37)** | 38.24 **(0.56)**
 :sparkles: :new: | **app/client/src/pages/Editor/gitSync/QuickGitActions/index.tsx** | **44.44** | **100** | **0** | **44.44**
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 50.47 **(-0.24)** | 37.72 **(-0.88)** | 36.21 **(0)** | 55.08 **(-0.27)**</details>

### CODE REVIEW

File: app/client/src/pages/Editor/IntegrationEditor/IntegrationsHomeScreen.tsx [hunk 1]
```diff
@@ -33,11 +33,13 @@ const HeaderFlex = styled.div`
 `;
 
 const ApiHomePage = styled.div`
+  display: flex;
+  flex-direction: column;
+
   font-size: 20px;
   padding: 20px 20px 0 20px;
   /* margin-left: 10px; */
-  min-height: calc(100vh - 66px);
```
Comment (Rishabh-Rathod) [inline app/client/src/pages/Editor/IntegrationEditor/IntegrationsHomeScreen.tsx:39]: Nitpick: Add `66px` as variable explaining what is refers. Is it is hieght of `bottomBar` ? currently it is not easily understandable.

File: app/client/src/pages/Editor/Popper.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (Rishabh-Rathod) [inline app/client/src/pages/Editor/Popper.tsx:1]: Why is eslint-disabled ?

**Review by Rishabh-Rathod — DISMISSED** [review]

File: app/client/src/pages/Editor/Popper.tsx (hunk 2 above)
Comment (rishabhsaxena) [reply app/client/src/pages/Editor/Popper.tsx:1]: typo

**Review by Rishabh-Rathod — APPROVED** [review]

File: app/client/src/pages/Editor/IntegrationEditor/IntegrationsHomeScreen.tsx (hunk 1 above)
Comment (rishabhsaxena) [reply app/client/src/pages/Editor/IntegrationEditor/IntegrationsHomeScreen.tsx:39]: Actually this one's a deletion

**Review by Rishabh-Rathod — DISMISSED** [review]

**Review by Rishabh-Rathod — APPROVED** [review]

**Pull request merged**